E2E: isolation test failure

The E2E isolation test appears to be failing during test setup:

```
=== RUN   TestChrootFS
=== RUN   TestChrootFS/testTaskEnvChroot
    utils.go:246: 
        	Error Trace:	/home/circleci/go/src/github.com/hashicorp/nomad/e2e/isolation/utils.go:246
        	            				/home/circleci/go/src/github.com/hashicorp/nomad/e2e/isolation/wait.go:64
        	            				/home/circleci/go/src/github.com/hashicorp/nomad/e2e/isolation/utils.go:228
        	            				/home/circleci/go/src/github.com/hashicorp/nomad/e2e/isolation/utils.go:221
        	            				/home/circleci/go/src/github.com/hashicorp/nomad/e2e/isolation/chroot_test.go:36
        	Error:      	Received unexpected error:
        	            	expected stopped alloc, but was: pending
        	Test:       	TestChrootFS/testTaskEnvChroot
        	Messages:   	failed to wait on alloc
--- FAIL: TestChrootFS/testTaskEnvChroot (57.63s)
=== RUN   TestChrootFS/testTaskImageChroot
    utils.go:246: 
        	Error Trace:	/home/circleci/go/src/github.com/hashicorp/nomad/e2e/isolation/utils.go:246
        	            				/home/circleci/go/src/github.com/hashicorp/nomad/e2e/isolation/wait.go:64
        	            				/home/circleci/go/src/github.com/hashicorp/nomad/e2e/isolation/utils.go:228
        	            				/home/circleci/go/src/github.com/hashicorp/nomad/e2e/isolation/utils.go:221
        	            				/home/circleci/go/src/github.com/hashicorp/nomad/e2e/isolation/chroot_test.go:60
        	Error:      	Received unexpected error:
        	            	expected stopped alloc, but was: pending
        	Test:       	TestChrootFS/testTaskImageChroot
        	Messages:   	failed to wait on alloc
--- FAIL: TestChrootFS/testTaskImageChroot (57.52s)
=== RUN   TestChrootFS/testDownloadChrootExec
    utils.go:246: 
        	Error Trace:	/home/circleci/go/src/github.com/hashicorp/nomad/e2e/isolation/utils.go:246
        	            				/home/circleci/go/src/github.com/hashicorp/nomad/e2e/isolation/wait.go:64
        	            				/home/circleci/go/src/github.com/hashicorp/nomad/e2e/isolation/utils.go:228
        	            				/home/circleci/go/src/github.com/hashicorp/nomad/e2e/isolation/utils.go:221
        	            				/home/circleci/go/src/github.com/hashicorp/nomad/e2e/isolation/chroot_test.go:84
        	Error:      	Received unexpected error:
        	            	expected stopped alloc, but was: pending
        	Test:       	TestChrootFS/testDownloadChrootExec
        	Messages:   	failed to wait on alloc
--- FAIL: TestChrootFS/testDownloadChrootExec (57.70s)
--- FAIL: TestChrootFS (172.98s)
```

--- a/e2e/isolation/pids_test.go
+++ b/e2e/isolation/pids_test.go
@@ -7,6 +7,7 @@ import (
 	"regexp"
 	"strings"
 	"testing"
+	"time"
 
 	"github.com/hashicorp/nomad/e2e/v3/cluster3"
 	"github.com/hashicorp/nomad/e2e/v3/jobs3"
@@ -62,7 +63,10 @@ func testExecHostPID(t *testing.T) {
 }
 
 func testExecNamespaceAllocExec(t *testing.T) {
-	job, cleanup := jobs3.Submit(t, "./input/alloc_exec.hcl")
+	job, cleanup := jobs3.Submit(t,
+		"./input/alloc_exec.hcl",
+		jobs3.Timeout(time.Second*30), // exec can be a bit slow
+	)
 	t.Cleanup(cleanup)
 
 	logs := job.Exec("group", "sleep", []string{"ps", "ax"})
@@ -76,6 +80,7 @@ func testJavaNamespacePID(t *testing.T) {
 	job, cleanup := jobs3.Submit(t,
 		"./input/java.hcl",
 		jobs3.WaitComplete("group"),
+		jobs3.Timeout(time.Second*30), // exec prestart can be slow
 	)
 	t.Cleanup(cleanup)
 
@@ -87,6 +92,7 @@ func testJavaHostPID(t *testing.T) {
 	job, cleanup := jobs3.Submit(t,
 		"./input/java_host.hcl",
 		jobs3.WaitComplete("group"),
+		jobs3.Timeout(time.Second*30), // exec prestart can be slow
 	)
 	t.Cleanup(cleanup)
 
@@ -97,7 +103,10 @@ func testJavaHostPID(t *testing.T) {
 }
 
 func testJavaNamespaceAllocExec(t *testing.T) {
-	job, cleanup := jobs3.Submit(t, "./input/alloc_exec_java.hcl")
+	job, cleanup := jobs3.Submit(t,
+		"./input/alloc_exec_java.hcl",
+		jobs3.Timeout(time.Second*30), // exec prestart can be slow
+	)
 	t.Cleanup(cleanup)
 
 	logs := job.Exec("group", "sleep", []string{"ps", "ax"})

